src/components/inputs/radio-input/radio-group.js

```diff
@@ -41,6 +41,13 @@ const Group = props => {
         onFocus: props.onFocus,
         onBlur: props.onBlur,
       });
+      return child.props.components.WrapperComponent ? (
```

Comment:
Nice! Wondering, as I always struggle to read `<a` with a lowercase character if it is actually a React component. Would 

```js
const Wrapper = child.props.components.WrapperComponent || React.Fragment`
```

work to then just do:

```js
<Wrapper>{children}</Wrapper>
```

Reply:
Hm, another approach we could go for just allows any `child` but leave it up to the consumer to not allow prop spreading. That way we don't need a new components prop. 

So for your case, you would do something like this. Basically capture the props in your wrapper then pass them to the component that should receive them (RadioOption).
```
const Wrapper = props => {
 <ToolTip off={isTooltipOff}>
    <RadioInput.Option {...props} />
  <ToolTip>
}
```

Reply:
I like this self-service approach. Would we be recursively searching for an RadioInput.Option as a child for the invariant, or removing the check all together, what do you think?

Reply:
I would be fine with removing it. A recursive search could also be misleading. As in: somewhere in the component tree there is a `Radio.Option` but it wasn't the one the developer but "nearby" his concern. So the invariant will maybe rarely give an appropriate hint.

Reply:
Hm, I'm not sure how we could do this though without making it a breaking change.